### PR TITLE
fixed fixmes and slightly refactored sitelink creation

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
depends on #51

### CODE REVIEW

File: tests/browser/features/support/modules/sitelink_module.rb [hunk 1]
```diff
@@ -99,19 +99,30 @@ def add_sitelinks(sitelinks)
...
     wait_for_api_callback
   end
 
+  def insert_site(index, site)
+    wait_until do
+      site_id_input_field(index).exists?
+    end
+    site_id_input_field_wd(index).clear
+    site_id_input_field_wd(index).send_keys site
+  end
```
Comment (zeljkofilipin) [inline tests/browser/features/support/modules/sitelink_module.rb]: I do not have the time to take a deeper look, but I am pretty sure the above method could be implemented something like

    def insert_site(element, site)
      element.when_present.set site
    end

**Comment by zeljkofilipin** [comment]
:+1: in general, see one inline comments

File: tests/browser/features/support/modules/sitelink_module.rb [hunk 2]
```diff
@@ -99,19 +99,30 @@ def add_sitelinks(sitelinks)
...
+
+  def insert_page(index, page)
+    wait_until do
+      page_input_field(index).exists?
+    end
+    page_input_field_wd(index).clear
+    page_input_field_wd(index).send_keys page
+  end
+
```
Comment (zeljkofilipin) [inline tests/browser/features/support/modules/sitelink_module.rb]: The same comment as above.

**Comment by WMDE-Fisch** [comment]
plz merge @tobijat

**Comment by tobijat** [comment]
Works!

**Pull request merged**